extensions/resteasy/deployment/src/test/java/io/quarkus/resteasy/test/NotFoundExceptionMapperTestCase.java

```diff
@@ -54,7 +53,6 @@ public void testJsonResourceNotFound() {
                 .contentType(ContentType.JSON);
     }
 
-    @Disabled("https://github.com/quarkusio/quarkus/issues/5424")
     @Test
     public void shouldDisplayNewAddedFileIn404ErrorPage() throws InterruptedException {
```

Comment:
nit: We can remove the `throws InterruptedException` from method signature signature